Examples/src/main/AndroidManifest.xml

```diff
@@ -42,14 +48,7 @@
                   android:screenOrientation="landscape"/>
 
         <activity android:name=".ArActivity"
```

Comment:
Hmm, could you please explain why launcher activity had changed?

Reply:
Actually I think to make all Kotlin activities as launchers with captions (we have different use cases). 
WDYT?

Reply:
Made all Kotlin classes launchers.

Reply:
Pls create an issue to add some nice toggle inside of an Examples app for future

Reply:
Let's leave toggling features for Teaser and preserve different entry points here.